services/attester/standard/attest.go

```diff
@@ -140,19 +153,28 @@ func (s *Service) attest(
 		return nil, errors.Wrap(err, "failed to sign beacon attestations")
 	}
 	s.log.Trace().Dur("elapsed", time.Since(started)).Msg("Signed")
+	var attestations []*spec.VersionedAttestation
 
-	attestations := s.createAttestations(ctx, duty, accounts, committeeIndices, validatorCommitteeIndices, committeeSizes, data, sigs)
-	if len(attestations) == 0 {
-		s.log.Info().Msg("No signed attestations; not submitting")
-		return attestations, nil
+	if epoch < s.electraForkEpoch {
+		attestations = s.createAttestations(ctx, duty, accounts, committeeIndices, validatorCommitteeIndices, committeeSizes, data, sigs)
+		if len(attestations) == 0 {
```

Comment:
This check is duplicated, you could move it outside of the separate branches.

You may also want to consider making the ifs a switch, which is tidier and allows for easier expansion whenever the next change happens.